src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
-// write code here
+const items = document.querySelectorAll('li');
+const list = document.querySelector('ul');
+
+function convertNumber(str) {
+  const newStr = str.slice(1);
+
+  return +newStr.split(',').join('');
+}
+
+function sortList(arr) {
+  const sortArr = Array.from(arr);
+
+  sortArr.sort((a, b) => {
+    const first = convertNumber(a.getAttribute('data-salary'));
```

Comment:
Add more descriptive names for variables
```suggestion
    const firstSalary = convertNumber(a.getAttribute('data-salary'));
```